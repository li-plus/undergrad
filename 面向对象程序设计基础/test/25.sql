create database oop;
use oop;
create table t(a int, b double, primary key(b)); 
insert into t(a, b) values(4, 1); 
insert into t(a, b) values(5, 2); 
insert into t(a, b) values(3, 3); 
insert into t(a, b) values(1, 4); 
insert into t(a, b) values(2, 5); 
select * from t order by a; 
/********************** Sample Output *************************
a       b       
1       4.0000  
2       5.0000  
3       3.0000  
4       1.0000  
5       2.0000  
**************************************************************/
select * from t order by b; 
/********************** Sample Output *************************
a       b       
4       1.0000  
5       2.0000  
3       3.0000  
1       4.0000  
2       5.0000  
**************************************************************/
drop database oop;