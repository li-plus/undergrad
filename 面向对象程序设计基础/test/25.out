a       b       
1       4.0000  
2       5.0000  
3       3.0000  
4       1.0000  
5       2.0000  
a       b       
4       1.0000  
5       2.0000  
3       3.0000  
1       4.0000  
2       5.0000  
